README.md

```diff
@@ -17,3 +17,14 @@ This repo includes:
 ### Nightly builds
 
 The latest built and tested packages from this repo are available on this MyGet feed: https://dotnet.myget.org/gallery/aspnetwebstack-dev
+
+### Tags and releases
+
+Git tag|Git branch|Other products|MVC package versions|Web API package (product) versions|Web Pages package versions
+--------|--------------|------------|------------|------------|------------
+[v2.0.4](https://github.com/aspnet/AspNetWebStack/tree/v2.0.4)|[v2.0-rtm](https://github.com/aspnet/AspNetWebStack/tree/v2.0-rtm)||4.0.40804|4.0.30506|2.0.30506
```

Comment:
The details pages in the NuGet gallery don't start distinguishing package and product versions for Web API until v5.0. Add "(2.0.3)" anyhow?